images/router/haproxy/conf/haproxy-config.template

```diff
@@ -334,17 +338,21 @@ backend be_tcp_{{$cfgIdx}}
       {{ if ne $weight 0 }}
         {{ with $serviceUnit := index $.ServiceUnits $serviceUnitName }}
           {{ range $idx, $endpoint := endpointsForAlias $cfg $serviceUnit }}
-            {{ with $healthIntv := index $cfg.Annotations "router.openshift.io/haproxy.health.check.interval" }}
-              {{ if (matchPattern "[1-9][0-9]*(us|ms|s|m|h|d)?" $healthIntv) }}
+            {{ if no $endpoint.NoHealthCheck }}
```

Comment:
F0818 18:30:04.056833       1 helpers.go:108] error: template: haproxy-config.template:294: function "no" not defined